toolchain/check/import_ref.cpp

```diff
@@ -70,6 +71,25 @@ auto AddImportRef(Context& context, SemIR::ImportIRInst import_ir_inst,
   return import_ref_id;
 }
 
+static auto AddLoadedImportRef(Context& context,
```

Comment:
Please add a comment saying why to use this function instead of e.g. `AddImportRef`, or how it is different